src/components/SimpleSearch.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './AccomodationSearch.css'
+import Box from '@mui/material/Box';
+import InputAdornment from '@mui/material/InputAdornment';
+import TextField from '@mui/material/TextField';
+import { KingBed } from '@mui/icons-material';
+import { MDBBtn } from 'mdb-react-ui-kit';
+
+
+const SimpleSearch = () => {
```

Comment:
You need to add state to the form and then have an `onSubmit` handler that logs the form's values